UPGRADE-3.1.md

```diff
@@ -24,6 +24,22 @@ FrameworkBundle
 
  * As it was never an officially supported feature, the support for absolute
    template paths has been deprecated and will be removed in Symfony 4.0.
+ * Since the refactoring of 2.8, form types without dependencies are autoloaded
+   by the `DependencyInjectionExtension` in the form registry.
+   Consequently they are deprecated and will be removed in 4.0.
+   You should use the fully-qualified name instead.
+
+   Before:
+
+   ```php
+   $type = $this->container->get('form.type.text');
+   ```
+
+   After:
+
+    ```php
+    $type = '\Symfony\Component\Form\Extension\Core\Type\TextType':
```

Comment:
@Tobion agreed, it misses the `new`.

Actually we could just remove them, but we have to deprecate them so devs who had the crazy idea to use them as services might be warned to do otherwise.

Telling them to instantiate the class instead can look trivial I agree, but may be needed.
I'm not so sure about both ways.

Waiting for a final call on this, thanks again for the review.